### ISSUE TITLE
Some distros use apache2ctl instead of apachectl

### ISSUE THREAD

**Issue by candrews** [icing/mod_h2#218]
Gentoo uses `apache2ctl` as the name of that utility, not `apachectl`, causing mod_h2 to fail to configure with this error:
```
configure: error: "not found: /usr/sbin/apachectl"
```
This problem was introduced by https://github.com/icing/mod_h2/commit/c6b148811cea82705fb4f2e7c14af1a9f443449c

A possible solutions to this problem is to add a `--with-apachectl` configure argument analogous to the existing `--with-apxs` one.

Reported downstream at https://bugs.gentoo.org/800908

**Comment by icing** [comment]
Did it work before that? I thought I had only fixed the path using `apxs` queries.

I saw that debian provides `apachectl` and `apache2ctl`, but I did not check other distros.

Maybe the simplest fix would be to check for both at test startup and use the one available.

Btw: since I made the docker setups in the project regarding debian-sid and archlinux, I would not mind merging in a gentoo setup as well! ;-)
